README.md

```diff
@@ -230,7 +230,7 @@ certificate.
 #### Generating a remote-issued certificate and private key.
 
 ```
-cfssl gencert -remote=remote_server [-hostname=comma,separated,hostnames] csrjson
+cfssl gencert -remote=remote_server [-hostname=comma,separated,hostnames] | csrjson
```

Comment:
This is incorrect, but I admit it is a little confusing. It would be better to put it "csr.json"